Sources/ContentScrollView.swift

```diff
@@ -83,40 +94,81 @@ open class ContentScrollView: UIScrollView {
         setContentOffset(.zero, animated: false)
     }
 
-    private func setupPages() {
+    /// setup empty containerView for `dataSource.numberOfPages(in:)`,
+    /// so that pageView is lazy added.
```

Comment:
*lazily

I like this approach. Still wasting memory with empty invisible views, but simple. 👍